### PR TITLE
TAO-8362 - Remove error code from waiting dialog

### PR DESCRIPTION
**Pull request by ferenckiss89**
Related ticket: https://oat-sa.atlassian.net/browse/TAO-8362

- error code + message is not being sent anymore in case of reconnection issue
- default value has been set for the message parameter in case the error message is not passed

### CODE REVIEW

File: views/js/runner/plugins/controls/connectivity/connectivity.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ define([
 
                         //creates the waiting modal dialog
                         dialog = waitingDialog({
-                            message : __('You are encountering a prolonged connectivity loss. ') + message,
+                            message : __('You are encountering a prolonged connectivity loss. '),
```
Comment (oatymart) [inline views/js/runner/plugins/controls/connectivity/connectivity.js]: We could keep the message here but only include it if it is a string (see below).

Comment (oatymart) [reply views/js/runner/plugins/controls/connectivity/connectivity.js]: Then, we could:
L198: remove the passing of the error message: `self.displayWaitingDialog()`
L177: keep passing the (could be informational) message: no change required

Comment (oatymart) [reply views/js/runner/plugins/controls/connectivity/connectivity.js]: In this way, the message would be gone only in the case of connectivity error. And the change to current behaviour is minimised.

Comment (ferenckiss89) [reply views/js/runner/plugins/controls/connectivity/connectivity.js]: Good point. Fixing it.

**Pull request merged**